pkg/types/args.go

```diff
@@ -26,8 +26,8 @@ import (
 type UnmarshallableBool bool
 
 // UnmarshalText implements the encoding.TextUnmarshaler interface.
-// Returns boolean true if the string is "1" or "[Tt]rue"
-// Returns boolean false if the string is "0" or "[Ff]alse"
+// Returns boolean true if the string is "1" or "[true]"
+// Returns boolean false if the string is "0" or "[flase]"
 func (b *UnmarshallableBool) UnmarshalText(data []byte) error {
 	s := strings.ToLower(string(data))
```

Comment:
Have you ever noticed this line? I think the annotations are right.

Reply:
> I haven't noticed this line before, I will pay attention to it later. Thank you!